inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -281,7 +281,7 @@ public function get_sitemap_links( $type, $max_entries, $current_page ) {
 
 				$stacked_urls[] = $url['loc'];
 
-				if ( (int) $post->ID === $this->get_page_for_posts_id() || (int) $post->ID === $this->get_page_on_front_id() ) {
+				if ( $post->ID === $this->get_page_for_posts_id() || $post->ID === $this->get_page_on_front_id() ) {
```

Comment:
Both `get_page_for_posts_id` and `get_page_on_front_id` are returning an integer. Removing the typehint will break this check.

Reply:
Oh wait. I was too quick. You are typehinting this below 👍